### PR TITLE
Change async/await command handling

### PR DESCRIPTION
**Pull request by d0ctr**

### CODE REVIEW

File: telegram/telegram-client.js [hunk 1]
```diff
@@ -293,21 +292,30 @@ class TelegramInteraction {
...
-                if (err) {
-                    return await this._reply(err, overrides);
-                }
-                if (response instanceof String || typeof response === 'string') {
-                    return await this._reply(response, overrides);
-                }
-                if (response instanceof Object) {
-                    return await this._replyWithMedia(response, overrides);
-                }
+                this.handler[this.command_name](this.context, this).then(async ([err, response, _, overrides]) => {
```
Comment (vladislavzasyadko) [inline telegram/telegram-client.js:296]: look like it could use some refactoring

**Comment by d0ctr** [comment]
@vladislavzasyadko, what would you suggest?

**Review by vladislavzasyadko — DISMISSED** [review]
async go brrrrrr

**Review by vladislavzasyadko — APPROVED** [review]
LGTM👍

**Comment by d0ctr** [comment]
Closes #23

**Pull request merged**